Assignment/solution_student/index.html

```diff
@@ -1,35 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
 
-Sample Content
-——-
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
 
-ABOUT ME PAGE
+<body>
 
-I ENJOY LIFE AS A DEVELOPER
+<hr>
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+<h3 class="navigation">About Me | Portfolio</h3>
 
+<h1>Joe Johnson</h1>
 
-RESUME PAGE
+<hr>
 
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+<img src="images/avatar.png" alt="Author Profile" class="avatar">
 
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
+<h3>I ENJOY LIFE AS A DEVELOPER</h3>
 
-Web Developer, CitiPix 2015 – present
-Built web app prototype that allows users to store and quickly retrieve photos using keywords.
+<p>I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
 
+<hr>
 
-EDUCATION
-
-General Assembly — FEWD 2015
-College of Soft Knocks — B.S. 2009
-School of Hard Knocks — 2005 (Honors)
+<ul class="social">
+	<li><a href="url">Facebook |</a>
+	<li><a href="url"> Twitter |</a>
+	<li><a href="url"> Instagram |</a>
+	<li><a href="url"> LinkedIn</a>
```

Comment:
Oops! Looks like you forgot the closing `<ul>` — make sure you close tags as you open them to avoid this happening.